### PR TITLE
Reduce the number of queues to workflow x lane

### PR DESCRIPTION
**Pull request by jcoyne**
Prior to deploying, we need to update and deploy configuration for:
- [x] was_robots (https://github.com/sul-dlss/was_robot_suite/pull/124)
- [x] gis_robots (https://github.com/sul-dlss/gis-robot-suite/pull/198)
- [x] preservation_robots (https://github.com/sul-dlss/preservation_robots/pull/138)
- [x] common-assembly (https://github.com/sul-dlss/common-accessioning/pull/327)

### CODE REVIEW

File: app/services/queue_service.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def initialize(step)
   # Enqueue the provided step
   def enqueue
     # Perform the enqueue to Resque
```
Comment (ndushay) [inline app/services/queue_service.rb]: This comment seems ... a little confused and not helpful?

File: app/services/queue_service.rb [hunk 2]
```diff
@@ -36,10 +36,10 @@ def step_name
   # Generate the queue name from step
```
Comment (ndushay) [inline app/services/queue_service.rb]: should "from step" be removed in this PR?

Comment (jcoyne) [reply app/services/queue_service.rb]: it is still being derived from the current step object.

**Comment by justinlittman** [comment]
This scheme should be described in README (either here or commons-accessioning).

**Pull request merged**